src/Contracts/Transporter.php

```diff
@@ -21,7 +22,7 @@ interface Transporter
      *
      * @throws ErrorException|UnserializableResponse|TransporterException
      */
-    public function requestObject(Payload $payload): array;
+    public function requestObject(Payload $payload, bool $stream = false): array|Generator;
```

Comment:
Can you somehow, make this method return a single object - for both cases - and have  the stream option sent on the Payload object?

Reply:
Created new `Response` object for `Transporter`. Stream option handling moved to `Payload` object.